doc/xml/reference.xml

```diff
@@ -488,6 +515,15 @@
                         <example>y</example>
                     </config-key>
 
+                    <!-- ======================================================================================================= -->
+                    <config-key id="repo-local" name="Repository Locality">
+                        <summary>Repository is local.</summary>
+
+                        <text>Indicates that the repository for the specified index is local and any repository host settings (e.g. <setting>repo1-host</setting>) should be ignored.</text>
```

Comment:
Maybe `and any repository host settings for that index` - just to ensure people don't think repo host settings for all repos are ignored.

Reply:
Fixed in commit 2023430cbc97bb5deaf0f28372ecaebc1bbce3f7

Reply:
Looks good to me.